src/index.html

```diff
@@ -4,24 +4,410 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
     />
     <title>Catalog</title>
+    <link
+      rel="stylesheet"
+      href="styles/index.scss"
```

Comment:
The `href` attribute is linking to an SCSS file (`styles/index.scss`). Browsers cannot directly interpret SCSS files, so you need to compile this SCSS into a CSS file and link to the resulting CSS file instead.